src/state/governance/hooks.ts

```diff
@@ -1,4 +1,5 @@
-import { PNG } from 'src/constants/tokens'
+
+import { PNG } from '../../constants/tokens'
```

Comment:
lets use absolute path here. please revert this change

Reply:
Updated by @jb1011 ✅